Contributing.md

```diff
@@ -2,6 +2,20 @@
 
 ## How to start development
 
+### Used Tools
+
+This project uses EditorConfig to standardize text editor configuration.
+Visit http://editorconfig.org for details.
+Visit https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig for the VSCode Plugin.
+
+This project uses TypeScript TSLint plugin for VSCode.
+Visit https://github.com/Microsoft/typescript-tslint-plugin for details.
+
+This project uses Mocha and Chai for testing.
+Visit http://mochajs.org and http://chaijs.com for details.
+
+Code coverage generated by http://istanbul.js.org
```

Comment:
unrelated to "adding test"

Reply:
keeping it here anyways because it improves the documentation